src/component.rs

```diff
@@ -24,3 +24,4 @@ pub const COMPONENT_PHP_MYSQLI_ID: i32 = 8004;
 pub const COMPONENT_PHP_PREDIS_ID: i32 = 8006;
 pub const COMPONENT_PHP_MEMCACHED_ID: i32 = 20;
 pub const COMPONENT_PHP_REDIS_ID: i32 = 7;
+pub const COMPONENT_RABBITMQ_PRODUCER_ID: i32 = 52;
```

Comment:
So, only the producer side? No consumer?

Reply:
The consumer plan will be added later (PHP is not very suitable as a consumer).